bin/crew

```diff
@@ -1300,33 +1314,11 @@ def install_package(pkgdir)
       end
     end
 
-    # check if the available rsync command support ACLs
+    install_files(File.join('.', HOME)) if Dir.exist?(File.join('.', HOME))
```

Comment:
```
install_files(File.join('.', HOME), HOME) if Dir.exist?(File.join('.', HOME))
```
?

Reply:
Actually, why don't we put the `Dir.exist?` logic inside the `install_files` function?  Something like:
```
if Dir.exist?(src)
  ...
else
  puts "#{src} directory does not exist.".lightred
end
```

Reply:
The existing setup is better, because the code block is used for BOTH `CREW_PREFIX` and `HOME` and there may be no `HOME` in the binary archive we are installing. There's no need to complain if that's the case.

Reply:
Adding the logic inside the function is more elegant because currently there is no way to validate the directory unless the conditional is outside the function and we definitely want to throw an error message for an invalid directory rather than the kludgy error we'll get with the first command that attempts to operate on it.

Reply:
But I see what you are saying about `HOME`.  It's fine to have both?

Reply:
I guess it makes sense to have a conditional throw an error if the function is passed a nonexistent `src` dir, and then also not invoke the function if the `src` dir doesn't exist.

Reply:
How about...
```
def install_files(prefix, path, dst = File.join( CREW_PREFIX, File.join(prefix, path).delete_prefix(".#{CREW_PREFIX}") ))
  src = File.join(prefix, path)
  if Dir.exist?(src)
    if File.executable?("#{CREW_PREFIX}/bin/crew-mvdir") && !CREW_DISABLE_MVDIR
      system "crew-mvdir #{@short_verbose} #{src} #{dst}", exception: true
    else
      warn 'crew-mvdir is not installed. Please install it with \'crew install crew_mvdir\' for improved installation performance'.yellow unless @pkg.name == 'crew_mvdir'
  
      if File.executable?("#{CREW_PREFIX}/bin/rsync") && system("#{CREW_PREFIX}/bin/rsync --version > /dev/null")
        # rsync src path needs a trailing slash
        src << '/' unless src.end_with?('/')
        system 'rsync', "-ah#{@verbose}HAXW", '--remove-source-files', src, dst, exception: true
      else
        system "cd #{src}; tar -cf - ./* | (cd #{dst}; tar -x#{@verbose}p --keep-directory-symlink -f -)", exception: true
      end
    end
  else
    puts "#{src} directory does not exist.".lightred
    return false 
  end
end
```

Reply:
(leaving aside the logic question about the parameters to this function.)